client/src/components/AllFeedback/index.js

```diff
@@ -20,21 +20,21 @@ const AllFeedback = ({ student }) => {
 	//Render the page on first load
 	useEffect(()=>{
 		updateFeedback();
-	}, []);
+	});
 
 	const feedbackToEdit = (feedback) => {
-		setShowComponent(true);
+		setIsEditing(true);
 		setFeedbackToBeEdited(feedback);
 	};
 
 	const noShowPage = () => {
```

Comment:
Can we change the name of this function to hide editing or something like that, please?

Reply:
Which one did you mean; feedbackToEdit or other state settings inside. However, it is not going to be an issue in my next commit